### DISCUSSION TITLE
50,000 Downloads — Call for Contributors 📣

### DISCUSSION THREAD

**Discussion by simonhamp** [NativePHP/laravel#409] [Announcements]
Hey team! Big news:

### NativePHP hit [50,000 installs](https://packagist.org/packages/nativephp/laravel/stats) earlier this week! 🤯 

This is an absolutely _phenomenal_ milestone!

I'm blown away by how many of you are using it to build really cool and interesting apps and games already and I know that many more are waiting patiently for that v1 release before they go full tilt into development.

Any of you who know me personally know that I don't get very animated, but I'm struggling to find the right words here to share just how excited I am about the potential for NativePHP's growth in 2025.

**No, this isn't the announcement to say that we're moving out of _alpha_ just yet... it's coming, but we just have a couple more pieces to get right first. Sorry!**

But we are closer than ever 🚀  Not much further to go now.

## We need your open source contributions!

NativePHP's existence and growth relies on two things: [**Funding**](https://nativephp.com/docs/1/getting-started/sponsoring) and **Open Source Contributions**.

If you're able to sponsor the project, that is always massively appreciated, but we'd like to encourage you to consider contributing in other ways too.

We're a positive, supportive and vibrant community of engineers at all levels and from all backgrounds. We want you to enjoy contributing to open source and supporting this project in this uniquely enjoyable way.

We want to make it as easy as possible for contributors to get involved in building NativePHP. The more of us there are, the faster things can move and the better the code and ecosystem will become, making for an easier, faster, and more reliable development experience as well as safer and richer apps for your clients, customers and users.

All we ask is that you have a positive attitude, keep an open mind, and contribute meaningfully with the best intentions for the wider community.

## How can you help?

While NativePHP may seem fairly complex on the surface, when it's all boiled down it's actually quite straightforward. If you're used to managing Laravel applications between a server and a web browser, you'll quickly get comfortable with NativePHP internals.

### Programming
If you've got experience with any/some/all of **PHP**, **Laravel**, **Electron**, **TypeScript**, or **Node**, then you can almost certainly get involved! And even if you're not there yet but you're prepared to learn some new stuff, then we'll be happy to guide you.

We've got plenty to do - building features, updating dependencies, fixing bugs, writing tests, improving debugging and development workflows - the list is endless! The more we do, the more we'll have to do.

### Testing
It doesn't just have to be writing code; we also need help with testing across all platforms - at least Windows, Mac and Linux for now.

Whether you're using one or all of these platforms, taking the time to test out builds and file bug reports is all considered contributing!

If you can find time to follow up with answers to maintainers questions and provide issue reproduction repos, this means so much to us and helps immensely.

### Documentation
Nobody's perfect. Despite the best of intentions, there are gaps and errors in our documentation. It always needs a careful eye. PRs that improve documentation are as valuable as - if not, more valuable than - feature PRs!

So if you spot mistakes, omissions, or you've found a good tidbit on an issue, discussion thread, or in Discord that you think deserves to live on the docs, get your PR-making fingers ready.

---

For the hundreds of you who have already been supporting NativePHP in this, please accept my eternal thanks - you have helped NativePHP get where it is today... and in many cases, continue to do so.

### Thank you 🙏🏼 

If you've not been involved yet but you're keen to support NativePHP in this important way, I'd love to help you get your wings!

Please join us on [Discord](https://discord.gg/X62tWNStZK), and feel free to chime in on [Issues](https://github.com/NativePHP/laravel/issues) and [Discussions](https://github.com/orgs/NativePHP/discussions).

If you're not sure where or how to begin, just ask. Someone will certainly be ready to help out.

Is there anything holding you back? What can we improve to make it easier for you or others to get involved?

Let's build the future... together

Simon

**Comment by dan1661** [comment]
i've been using laravel for many years, but never have experience in projects as contributor. let me know if any way I can help on this project. Thanks for the great work
